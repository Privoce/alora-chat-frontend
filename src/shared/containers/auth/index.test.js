import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	AuthContainer
} from 'shared/containers';

describe('<AuthContainer />', () => {
	const modelI = {
		title: 'Teste title AuthContainer',
		formContainer: (
			<div>
				<span>
					{
						'teste'
					}
				</span>
			</div>
		),
		footerInfo: (
			<span>
				{
					'info footer'
				}
			</span>
		),
		footerInfoLink: (
			<a href='#teste'>
				{
					'footer link'
				}
			</a>
		)
	};

	const getWrapper = (model) => {
		return shallow((
			<AuthContainer {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
