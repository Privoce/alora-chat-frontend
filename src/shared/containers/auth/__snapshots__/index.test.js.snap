// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthContainer /> renders correctly 1`] = `
<section
  className="auth-wrapper"
>
  <div
    className="form-container"
  >
    <h1
      className="title"
    >
      Teste title AuthContainer
    </h1>
    <div>
      <span>
        teste
      </span>
    </div>
    <div
      className="info-container"
    >
      <span>
        info footer
      </span>
      <a
        href="#teste"
      >
        footer link
      </a>
    </div>
  </div>
</section>
`;
